Javascript error after build

<img width="428" alt="screen shot 2016-01-20 at 12 59 59 pm" src="https://cloud.githubusercontent.com/assets/16512835/12461388/d64910ea-bf75-11e5-9ab3-8ecd3151b75d.png">

OS X 10.11.1

Performed clone from repo, installed via NPM (no errors, I think), created .app and then built the .dmg - but no dice.

Do I just repeat the install/build process?

`
Uncaught Exception:
Error: Cannot find module '/Applications/Brave.app/Contents/Resources/app/node_modules/sqlite3/lib/binding/node-v47-darwin-x64/node_sqlite3.node'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Applications/Brave.app/Contents/Resources/app/node_modules/sqlite3/lib/sqlite3.js:4:15)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)`

--- a/README.md
+++ b/README.md
@@ -19,7 +19,7 @@ Laptop and Desktop browser for OS X, Windows and Linux.
 
         cd browser-laptop
 
-3. Install the Node dependencies:
+3. Install the Node (v5+) dependencies:
 
         npm install
 

